packages/xs/xapi-idl.1.90.1/opam

```diff
@@ -10,10 +19,11 @@ remove: [
 depends: [ "uri"
            "re"
            "cmdliner"
-           "cohttp" {>= "0.15.2"}
+           "cohttp"
            "xmlm"
-           "rpc" {>= "1.9.4"}
+           "rpc" {>= "1.5.3"}
```

Comment:
Thanks for pointing out the dependency versions though, looking at the diff again I found a problem with the original `opam` file of `xcp-idl` (`cohttp` is duplicated in the `depends` section).